homeassistant/components/sensor/usps.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
@happyleavesaoc change request have the same affect like before the update on `__init__`. Please change it. You can also use the new https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/exceptions.py#L31 on `platform_setup` to indicate that hass should try it later for setups.